external/fv3fit/fv3fit/pytorch/graph_config.py

```diff
@@ -0,0 +1,64 @@
+import dataclasses
+from typing import Callable
+import torch.nn.functional as F
+import torch
+import torch.nn as nn
+from dgl.nn.pytorch import SAGEConv
+
+
+@dataclasses.dataclass
+class GraphNetworkConfig:
+    """
+    Attributes:
+        in_feats: number of input features
+        out_feats: number of output features
+        n_hidden: number of hidden channels
```

Comment:
Is this the total number of hidden channels, or the maximum number of channels in a u-net block?